cli/command_repository_set_parameters.go

```diff
@@ -43,7 +43,7 @@ func (c *commandRepositorySetParameters) setup(svc appServices, parent commandPa
 
 	cmd.Flag("max-pack-size-mb", "Set max pack file size").PlaceHolder("MB").IntVar(&c.maxPackSizeMB)
 	cmd.Flag("index-version", "Set version of index format used for writing").IntVar(&c.indexFormatVersion)
-	cmd.Flag("retention-mode", "Set the blob retention-mode for supported storage backends.").EnumVar(&c.retentionMode, "none", blob.Governance.String(), blob.Compliance.String())
+	cmd.Flag("retention-mode", "Set the blob retention-mode for supported storage backends.").EnumVar(&c.retentionMode, "none", blob.Governance.String(), blob.Compliance.String(), blob.Locked.String())
```

Comment:
If you mean https://kopia.io/docs/reference/command-line/common/repository-sync-to-azure/ maybe add some `destination-retention-mode`

Reply:
I guess storage agnostic is awkward since users will already have their configs set to S3 settings unless we fail on the putblobs & force them to update the settings to more general like 'MINIMAL' / 'MAXIMUM' or similar but then the docs would need to specify what each name meant for each provider